gradle/fork.gradle.kts

```diff
@@ -10,22 +10,28 @@ configure<ForkExtension> {
                 "projectLabel" to { description = "Nice project name (human-readable)" },
                 "projectGroup" to {
                     description = "Java package in source code and artifact 'group' coordinate"
-                    validator { javaPackage(); notContains("projectName") }
+                    validator { javaPackage(); notEndsWith("projectName") }
                 },
-                "aemInstanceType" to {
-                    select("local", "remote")
-                    description = "local - instance will be created on local file system.\nremote - connecting to remote instance only."
-                },
-                "aemInstanceRunModes" to { text("nosamplecontent") },
-                "aemInstanceJvmOpts" to { text("-server -Xmx1024m -XX:MaxPermSize=256M -Djava.awt.headless=true") },
                 "aemInstanceAuthorHttpUrl" to {
                     url("http://localhost:4502")
+                    optional()
                     description = "URL for accessing AEM author instance"
                 },
                 "aemInstancePublishHttpUrl" to {
                     url("http://localhost:4503")
+                    optional()
                     description = "URL for accessing AEM publish instance"
-                }
+                },
+                "aemInstanceType" to {
+                    select("local", "remote")
+                    description = "local - instance will be created on local file system\nremote - connecting to remote instance only"
+                    controller { toggle(value == "local", "aemInstanceRunModes", "aemInstanceJvmOpts", "aemLocalInstance*") }
```

Comment:
Remove "*" from prop name

Reply:
nope, it is intentional :) it will be dynamically expanded to `aemLocalInstanceJarUri`, `aemLocalInstanceLicenseUri` and `aemLocalInstanceZipUri` ;) it is a shorthand ;)